src/test/kotlin/study/domain/Language.kt

```diff
@@ -0,0 +1,3 @@
+package study.domain
+
+data class Language(val language: String, val level: Int)
```

Comment:
```suggestion
data class Language(private val language: String, private val level: Int)
```
private이어도 문제 없어보입니다 :)